ci-operator/step-registry/osac-project/cluster-tool/boot/osac-project-cluster-tool-boot-commands.sh

```
@@ -0,0 +1,335 @@
+echo "Waiting for SSH to be ready..."
+for i in $(seq 30); do
+    ssh -F "${SHARED_DIR}/ssh_config" ci_machine hostname 2>/dev/null && break
+    echo "  attempt ${i}/30 - retrying in 10s..."
+    sleep 10
+done
+
+# === Build merged pull-secret with CI registry credentials ===
+# /etc/pull-secret/.dockerconfigjson is auto-mounted in every CI pod
+# and contains credentials for registry.buildXX.ci.openshift.org
+echo "Building pull secret..."
+echo "  Cluster profile registries: $(jq -r '.auths | keys | join(", ")' ${CLUSTER_PROFILE_DIR}/pull-secret 2>/dev/null || echo 'PARSE ERROR')"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)